packages/preview-service/tests/testKnex.ts

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable camelcase */
+require('ts-node/register') //required to allow knex to understand typescript migration files
```

Comment:
Weird, you shouldn't need to do this. If knex migrations are ran by vitest, it should already be running w/ ts-node (or the equivalent).

Also - we probably want to do the same thing we do in `server` and only allow built .js migrations (in dist folder) to be ran, so that you can't accidentally run the same migration twice - once as a ts file with ts-node, and the 2nd time when its ran through dist.